<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paramsKPCA</class>
 <widget class="QWidget" name="paramsKPCA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelkernel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kernel Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="kernelTypeCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>103</y>
     <width>102</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Polynomial: (&amp;lt;x,y&amp;gt; + offset)^degree&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;RBF: exp(-&amp;lt;x,y&amp;gt;*width)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Sigmoid: tanh(&amp;lt;x,y&amp;gt; + offset)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Polynomial</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RBF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sigmoid</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelWidth_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>13</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Projected Dims</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="contourButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Eigenvector Iso-Lines</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kernelDegSpin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>103</y>
     <width>46</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Width of the kernel (gamma)
RBF and Polynomial only</string>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kernelWidthSpin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>103</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;RBF: Width of the kernel (gamma)&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Poly, Sigmoid: Offset (inhomogeneous kernel)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="param1Label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Degree</string>
   </property>
  </widget>
  <widget class="QLabel" name="param2Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="dimCountSpin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>44</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;# of dual eigen-vectors to be computed explicitly (will determine the dimensionality of the projected data)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="eigenGraph">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>2</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>K-PCA Eigenvalues</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="eigenList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>111</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
